### ISSUE TITLE
Kill and remove multiple containers in one go.

### ISSUE THREAD

**Issue by ahab94** [opencontainers/runc#1652]
There should be a mechanism of killing and removing multiple containers in one go.

Like in docker -> 
```Bash
docker rm -f some-id-1 some-id-2 some-id3
```

Which lets us do ->

```Bash 
docker rm -f $(docker ps -a -q)
```
at this moment this can be achieved by the following workaround

```Bash
runc list -q | while read -r line; do runc delete -f "$line"; done 
```

**Comment by cyphar** [comment]
We used to have this feature a while ago #1053, but it was removed. The justification for removing it is that having multi-delete means that (as a user) if `runc delete -f A B C D` fails then trying to figure out which containers failed to be deleted and what the current status is gets complicated. Effectively, allowing multi-container delete means that `runc delete -f` is no longer an "atomic" operation.

I'm not going to wax lyrical about POSIX, but if you read the POSIX entry for `rm` you'll notice that they have a *lot* of language about how to handle multiple-argument `rm` failures (and POSIX basically guarantees nothing about the failure state of `rm` with multiple arguments).
